webpack.common.config.js

```diff
@@ -68,17 +69,22 @@ module.exports = {
     }),
 
     new webpack.DefinePlugin({
-      // This is a temporary fix for okta-auth-js not populating
-      // this when consumed via npm
-      SDK_VERSION: JSON.stringify(0),
-      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken'),
-      DEFAULT_POLLING_DELAY: 500,
-      FRAME_ID: JSON.stringify('okta-oauth-helper-frame')
+      WIDGET_VERSION: JSON.stringify(packageJson.version)
```

Comment:
@alexma-okta yeah, I had the same question when I first saw this. Turns out it's a webpack thing - https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin.